components/Markdown/vendor-patched/symfony/yaml/composer.json

```diff
@@ -1,41 +1,40 @@
 {
-	"name": "symfony/yaml",
-	"type": "library",
-	"description": "Loads and dumps YAML files",
-	"keywords": [],
-	"homepage": "https://symfony.com",
-	"license": "MIT",
-	"authors": [
-		{
-			"name": "Fabien Potencier",
-			"email": "fabien@symfony.com"
-		},
-		{
-			"name": "Symfony Community",
-			"homepage": "https://symfony.com/contributors"
-		}
-	],
-	"require": {
-		"php": ">=8.2",
-		"symfony/deprecation-contracts": "^2.5|^3.0",
-		"symfony/polyfill-ctype": "^1.8"
-	},
-	"require-dev": {
-		"symfony/console": "^6.4|^7.0"
-	},
-	"conflict": {
-		"symfony/console": "<6.4"
-	},
-	"autoload": {
-		"psr-4": {
-			"Symfony\\Component\\Yaml\\": ""
-		},
-		"exclude-from-classmap": [
-			"/Tests/"
-		]
-	},
-	"bin": [
-		"Resources/bin/yaml-lint"
-	],
-	"minimum-stability": "dev"
+    "name": "symfony/yaml",
+    "type": "library",
+    "description": "Loads and dumps YAML files",
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/deprecation-contracts": "^0.0.11",
```

Comment:
Changing nested composer.json files is undesired and needs to be resolved before merging